staging/src/k8s.io/apiserver/pkg/authentication/config/v1alpha1/types.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type AuthenticationConfiguration struct {
+	metav1.TypeMeta
+
+	// jwt is a list of OIDC providers to authenticate Kubernetes users.
```

Comment:
I think we need to come up with a way to describe this without using OIDC since we really only use that for public key discovery.

Reply:
Use text from https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens and updated! LMK what you think.